lib/crypto.ts

```diff
@@ -0,0 +1,32 @@
+const crypto = require('crypto')
+const Secp256k1 = require('@lionello/secp256k1-js')
+
+import { KeyPair } from 'types'
+
+export function generateKeypair(): KeyPair {
+  let publicKey = null
+  let privateKey = null
+
+  do {
+    const privateKeyBuf = crypto.randomBytes(32)
+    privateKey = Secp256k1.uint256(privateKeyBuf, 16)
+    publicKey = Secp256k1.generatePublicKeyFromPrivateKeyData(privateKey)
+  } while (
+    !publicKey ||
+    privateKey.gte(
```

Comment:
Is this supposed to be `.get()`?

Reply:
Don't think this is causing the 31 bytes error but trying look see why that might happen

Reply:
Nope, gte is correct, I copied this from our existing `lib/user.ts` file which has been removed in this PR. I think it stands for greater than or equal to. I'll have a look at this code again because I think it can be simplified.